Ansible/examples/host_vars/xray/hosts.yml

```diff
@@ -0,0 +1,18 @@
+---
+all:
+    vars:
+        ansible_user: "centos"
+    children:
+        xray:
+            vars:
+                xray_version: 3.3.0
+                jfrog_url: http://ec2-18-210-33-94.compute-1.amazonaws.com
+                master_key: "c97b862469de0d94fbb7d48130637a5a"
+                join_key: "9bcca98f375c0728d907cc6ee39d4f02"
+                db_type: "postgresql"
+                db_driver: "org.postgresql.Driver"
+                db_url: "postgres://10.0.0.5:5432/xraydb?sslmode=disable"
+                db_user: "xray"
+                db_password: "xray"
+            hosts:
+                3.17.132.222
```

Comment:
hard coded ip?

Reply:
This is fine. It is just an example file.